grafana/resource_cloud_stack.go

```diff
@@ -80,6 +80,9 @@ Changing region will destroy the existing stack and create a new one in the desi
 				Optional:    true,
 				Default:     true,
 				Description: "Whether to wait for readiness of the stack after creating it. The check is a HEAD request to the stack URL (Grafana instance).",
+				// Suppress the diff if the new value is "false" because this attribute is only used at creation-time
+				// If the diff is suppress for a "true" value, the attribute cannot be read at all
+				DiffSuppressFunc: func(k, oldValue, newValue string, d *schema.ResourceData) bool { return newValue == "false" },
```

Comment:
IIRC it's possible not to name the parameters that we won't use, as:
```suggestion
				DiffSuppressFunc: func(_, _, newValue string, _ *schema.ResourceData) bool { return newValue == "false" },
```

Is this a nice feature or not? I can't say 😬